### DISCUSSION TITLE
Looking for ideas to implement an specific chart

### DISCUSSION THREAD

**Discussion by rezafaraji93** [patrykandpatrick/vico#333] [Questions] [answered]
First of all, thanks for this great library. I am here to ask you for help and your ideas to implement such a chart:
![Screenshot 2023-08-03 at 1 44 53 PM](https://github.com/patrykandpatrick/vico/assets/71515156/c2d67624-1822-4a77-a08a-5d5cb3d7b898)

here I have to always show zero on the chart.
this is my values list:
`val cpnlList = persistentListOf(
    0.0,
    -2012196.764386328,
    693348.5028034318,
    -7028607.032712051,
    665917.1975951893,
    -1298816.0578850303,
    -6256152.498508661,
    -4724717.733868127,
    -3721726.9737097495,
    -6695485.619791051,
    -5341153.389804348,
    -5409470.417389605,
    -7705668.317909865,
    -2435555.822211829,
    109386.38438295759,
    -3846129.3965776404,
    -6025766.503346017,
    2082134.9030501721,
    -212756.3867533505,
    -5570536.721019392,
    -2703216.0075408597,
    -2488336.4886773773,
    658926.3167034662,
    -2014046.2867599824,
    -6200383.5159033025,
    -5533474.407293849,
    201257.65863157436,
    -4009867.7381325625,
    -5759134.599792847
)`

and this is my bottom axis values list:
`val dates = persistentListOf(
    "2023-06-30",
    "2023-07-01",
    "2023-07-02",
    "2023-07-03",
    "2023-07-04",
    "2023-07-05",
    "2023-07-06",
    "2023-07-07",
    "2023-07-08",
    "2023-07-09",
    "2023-07-10",
    "2023-07-11",
    "2023-07-12",
    "2023-07-13",
    "2023-07-14",
    "2023-07-15",
    "2023-07-16",
    "2023-07-17",
    "2023-07-18",
    "2023-07-19",
    "2023-07-20",
    "2023-07-21",
    "2023-07-22",
    "2023-07-23",
    "2023-07-24"
)
`

**Comment by rezafaraji93** [comment]
The problem here is that I can not specify the red color for minus percents. And also I could not find a way to put the 0% value on the middle of the y axis.

**Reply by Gowsky** [reply, marked as answer]
Hello @rezafaraji93, the current version of Vico may not allow you to easily recreate the chart from the sample. The upcoming 1.9.0 release will improve how vertical axes handle charts with positive and negative values, so the `y=0` will be always visible.
Regarding distinct colors for positive and negative values — we will support them in future releases. However, you can work around this limitation by splitting your single list of entries into two. One for positive, and a second one for negative values. Then you will need to set two different `LineComponent`s to your column chart, and the `mergeMode` to `MergeMode.Stack`.

**Reply by patrickmichalik** [reply]
Hello! [Vico 1.9.0](https://github.com/patrykandpatrick/vico/releases/tag/v1.9.0), which @Gowsky mentioned, is now available.
